docs/source/api.rst

```diff
@@ -111,6 +111,7 @@ type functions in the algorithms API but can be run with a
 .. autosummary::
    :toctree: stubs
 
+   retworkx.complement
```

Comment:
You also need to add the per type variants to the specific graph type methods general algorithms toctree above

Reply:
Done